.drone.yml

```
@@ -106,6 +90,12 @@ platform:
   arch: amd64
 
 steps:
+  - name: Build theme files
```

Not just the theme really
```suggestion
  - name: Build JS/CSS
```